src/Pages/home/Components/Title.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import brainImg from "../../../assets/brain_welcome.png";
+import AddIcon from "../../../assets/add.png";
+
+//todo Style
+const useStyle = makeStyles(() =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+    },
+
+    //! Box
+    boxWelcome: {
+      width: "616px", //decrease 15%
```

Comment:
I do not suggest using constant value. You can try to calculate width on Figma to a percentage.

Reply:
**Note**
Width property only, height should be constant.